contxt/services/bus.py

```diff
@@ -21,7 +21,9 @@ class MessageBusService(ConfiguredApi):
         ),
     )
 
-    def __init__(self, auth: Auth, organization_id: str, env: str = "production", **kwargs) -> None:
+    def __init__(self, auth: Auth, organization_id: str, custom_envs: List[ApiEnvironment] = None, env: str = "production", **kwargs) -> None:
+        if custom_envs:
+            MessageBusService._envs = custom_envs
```

Comment:
modifying a static class variable in an instance's initializer feels off, can consumers just set `MessageBusService._envs` once directly? that was the intention, ie:

```python
# Customize your environment
MessageBusService._envs += (...)

service = MessageBuseService()
```

Reply:
good call -- subclass made way more sense. thanks!

Reply:
awesome